src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -1,16 +1,25 @@
 from enum import Enum
 from typing import Optional
 
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID, extract_use_case_id
+
 
 class AggregationOption(Enum):
     HIST = "hist"
+    TEN_SECOND = "ten_second"
 
 
 METRIC_ID_AGG_OPTION = {
     "d:transactions/measurements.fcp@millisecond": AggregationOption.HIST,
     "d:transactions/measurements.lcp@millisecond": AggregationOption.HIST,
 }
 
+USE_CASE_AGG_OPTION = {UseCaseID.CUSTOM: AggregationOption.TEN_SECOND}
+
 
 def get_aggregation_option(metricId: str) -> Optional[AggregationOption]:
+    use_case_id: UseCaseID = extract_use_case_id(metricId)
+    if use_case_id in USE_CASE_AGG_OPTION:
+        return USE_CASE_AGG_OPTION[use_case_id]
+
     return METRIC_ID_AGG_OPTION.get(metricId)
```

Comment:
I think we should maybe do something a little more complex here, if a metric has aggregation option specified at a metricId level, it should override the aggregation option specified at use case level. Right now the behaviour is opposite.

Reply:
Ah, you're right. Let me do that